src/components/molecules/Visualizer/hooks.ts

```diff
@@ -259,6 +300,7 @@ export default ({
     shownPluginModalInfo,
     pluginPopupContainerRef,
     shownPluginPopupInfo,
+    isMobile: viewport?.isMobile,
```

Comment:
```suggestion
    isMobile: viewport?.isMobile ?? false,
```

Reply:
After discuss with red I added a default value (as 1080p) for viewport so that `viewport.isMobile` will not be `undefined` now.